bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -675,5 +675,4 @@ spec:
         {{- if .Values.persistence.selector }}
         selector: {{- include "common.tplvalues.render" (dict "value" .Values.persistence.selector "context" $) | nindent 10 }}
         {{- end -}}
-        {{- include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) | nindent 8 }}
```

Comment:
Acording to the template defined in [here](https://github.com/bitnami/charts/blob/a9077e0501cbc9f4460b91866ced3bd4d789ecff/template/CHART_NAME/templates/statefulset.yaml), I'd remove the lines 669-671: https://github.com/bitnami/charts/blob/e909092a93db4ecd386aa88439fd98cc7887eb79/bitnami/postgresql-ha/templates/postgresql/statefulset.yaml#L669-L671